PayPal/src/main/java/com/braintreepayments/api/paypal/PayPalCheckoutRequest.kt

```diff
@@ -66,6 +66,9 @@ import org.json.JSONObject
  *
  * @property contactPreference Preference for the contact information section within the payment flow. Defaults to
  * [PayPalContactPreference.NO_CONTACT_INFORMATION] if not set.
+ *
+ * @property amountBreakdown Breakdown of items associated to the total cost - this property is
+ * for checkout with vault requests only.
```

Comment:
```suggestion
 * @property amountBreakdown Breakdown of items associated to the total cost
```
This property is for any checkout or checkout with vault flow